ts/screens/PinLoginScreen.tsx

```diff
@@ -136,6 +132,8 @@ class PinLoginScreen extends React.Component<Props> {
           </Button>
           <View spacer={true} />
           <Text white={true}>{I18n.t("pin_login.pin.reset.tip")}</Text>
+          <View spacer={true} />
+          <CancelPaymentLink navigation={this.props.navigation} />
```

Comment:
how comes this button is always rendered? there should be a props that enables it when in the context of a payment

Reply:
I know. The PR is WIP missing the implementation of the control you mentioned